WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewViewModel.swift

```diff
@@ -38,13 +52,32 @@ final class ThemesPreviewViewModel: ObservableObject {
     func setSelectedPage(page: WordPressPage) {
         selectedPage = page
     }
+
+    @MainActor
+    func installTheme() async throws {
```

Comment:
Nit: should we rename this to `confirmSelection` since we're not installing the theme immediately for store creation?

Reply:
Good point. Renamed into `confirmThemeSelection` in https://github.com/woocommerce/woocommerce-ios/pull/11497/commits/a2c5244035e17ffc1fd7101594b4a38dff9d2f11